Resources/Textures/_Harmony/Clothing/Uniforms/Jumpskirt/hopmesskit.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV atcommithttps://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```

* Multiple spaces missing in copyright
```suggestion
  "copyright": "Made by TadJohnson00 and Timemaster99 for DeltaV at commit https://github.com/DeltaV-Station/Delta-v/pull/1501, resprited by CrimsonJupiter for Harmony.",
```
* `at commit https://github.com/DeltaV-Station/Delta-v/pull/1501` PR request is not a commit. Find a commit it was merged to master in.